interface/ceed-operator.c

```diff
@@ -1492,16 +1492,46 @@ int CeedOperatorView(CeedOperator op, FILE *stream) {
 
     for (CeedInt i = 0; i < num_suboperators; i++) {
       has_name = sub_operators[i]->name;
-      fprintf(stream, "  SubOperator %" CeedInt_FMT "%s%s:\n", i, has_name ? " - " : "", has_name ? sub_operators[i]->name : "");
-      CeedCall(CeedOperatorSingleView(sub_operators[i], 1, stream));
+      fprintf(stream, "  SubOperator %" CeedInt_FMT "%s%s%s\n", i, has_name ? " - " : "", has_name ? sub_operators[i]->name : "", is_full ? ":" : "");
+      if (is_full) CeedCall(CeedOperatorSingleView(sub_operators[i], 1, stream));
     }
   } else {
     fprintf(stream, "CeedOperator%s%s\n", has_name ? " - " : "", has_name ? op->name : "");
-    CeedCall(CeedOperatorSingleView(op, 0, stream));
+    if (is_full) CeedCall(CeedOperatorSingleView(op, 0, stream));
   }
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief View a `CeedOperator`
+
+  @param[in] op     `CeedOperator` to view
+  @param[in] stream Stream to write; typically `stdout` or a file
+
+  @return Error code: 0 - success, otherwise - failure
+
+  @ref User
+**/
+int CeedOperatorView(CeedOperator op, FILE *stream) {
+  CeedCall(CeedOperatorView_Core(op, stream, true));
+  return CEED_ERROR_SUCCESS;
+}
+
+/**
+  @brief View a brief summary `CeedOperator`
+
+  @param[in] op     `CeedOperator` to view brief summary
+  @param[in] stream Stream to write; typically `stdout` or a file
+
+  @return Error code: 0 - success, otherwise - failure
+
+  @ref User
+**/
+int CeedOperatorTerseView(CeedOperator op, FILE *stream) {
```

Comment:
Maybe `CeedOperatorViewTerse` is more inline with naming conventions? Since this is a special case of the `View` function?

Reply:
Eh, I just saw that `CeedOperatorSingleView` exists, so maybe what you have is more inline with naming conventions.

Reply:
Sure